adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -262,6 +262,28 @@ public BrowserInfo updateBrowserInfoFromRequest(BrowserInfo browserInfo, Payment
         return browserInfo;
     }
 
+    public ApplicationInfo updateApplicationInfoEcom(ApplicationInfo applicationInfo) {
+        updateApplicationInfo(applicationInfo);
+
+        CommonField adyenPaymentSource = new CommonField();
+        adyenPaymentSource.setName(PLUGIN_NAME);
+        adyenPaymentSource.setVersion(PLUGIN_VERSION);
+        applicationInfo.setAdyenPaymentSource(adyenPaymentSource);
+
```

Comment:
missing here:
```
CommonField merchantApplication = new CommonField();
        merchantApplication.setName(PLUGIN_NAME);
        merchantApplication.setVersion(PLUGIN_VERSION);
        applicationInfo.setMerchantApplication(merchantApplication);
```

Reply:
are we adding this in this PR or should we create another one?

Reply:
We are adding it in this.